Sources/System/Internals/WindowsSyscallAdapters.swift

```diff
@@ -621,14 +621,15 @@ fileprivate struct DecodedOpenFlags {
     }
 
     dwDesiredAccess = 0
-    if (oflag & _O_RDONLY) != 0 {
+    switch (oflag & (_O_RDONLY|_O_WRONLY|_O_RDWR)) {
```

Comment:
This probably deserves a comment to indicate that the flags are non-overlapping.

Reply:
Done.